flask_cors/__init__.py

```diff
@@ -150,6 +153,7 @@ def wrapped_function(*args, **kwargs):
             options.update(_get_app_kwarg_dict())
             options.update(_options)
             _serialize_options(options)
+            logger.debug('Final CORS options: {}'.format(str(options)))

```

Comment:
Correct me if I'm wrong, but decorator will not call init_app() on it's own, so it is possible to initiate CORS on specific route without instantinating CORS() class. In such case logging information will be lost.